### ISSUE TITLE
Force blur stops working while moving/resizing windows with wobbly windows turned on

### ISSUE THREAD

**Issue by pluiedev** [taj-ny/kwin-effects-forceblur#92]
On Plasma 6.1.3 w/ Wayland :(

https://github.com/user-attachments/assets/c575179c-a274-4c82-af5d-a2b2965f5a19

**Comment by taj-ny** [comment]
I got it to work, but it may take me a while to find and fix all of the bugs. I'll also have to include a fork of wobbly windows, as copying the header and changing all private members to public is probably a bad idea.

https://github.com/user-attachments/assets/b945a955-e783-4040-be88-29640fa17544

**Comment by EatedPantsu** [comment]
I'm just piggy-backing on this one, as it seems a bit related:

When minimizing a transparent window that is using effect to transform it while minimizing (such as magic lamp effect or scale) the minimized windows looses blur for a moment before minimizing.

[Screencast_20240809_131011.webm](https://github.com/user-attachments/assets/c51b6bad-157a-4480-9ee7-d49143be72bf)

**Comment by dylanmartinezs** [comment]
Hello! Is there any ETA for this?

**Comment by taj-ny** [comment]
> Hello! Is there any ETA for this?

No, there are more important things I need to do with this project and I'm too lazy to do them. The prototype I showed before was an ugly hack that duplicated a lot of code and required the window geometry to be calculated twice (once by the wobbly windows effect and another time by the blur effect).

**Comment by LuorixDev** [comment]
Is there any progress?
